pkg/core/dump.go

```diff
@@ -24,3 +28,55 @@ func Dump(bc *Blockchain, w *io.BinWriter, start, count uint32) error {
 	}
 	return nil
 }
+
+// Restore restores blocks from provided reader.
+// f is called after addition of every block.
+func Restore(bc *Blockchain, r *io.BinReader, skip, count uint32, f func(b *block.Block) error) error {
```

Comment:
This also shouldn't be a part of the `core`.

Reply:
Fixed.